2024/src/components/LanguageSwitch.tsx

```diff
@@ -50,7 +54,7 @@ export function LanguageSwitch(props: Props) {
           <React.Fragment key={langKey}>
             <Lang
               selected={selected}
-              href={!hasMounted || selected ? undefined : "#"}
+              // href={!hasMounted || selected ? undefined : "#"}
```

Comment:
Commented-out code tends to become incompatible over time. Can you please just delete it.

Reply:
Deleted :)